src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -707,3 +706,20 @@ export class NotebookService extends Disposable implements INotebookService {
 		this._onCodeCellExecutionStart.fire();
 	}
 }
+
+/**
+ * @param notebookUri of the notebook
+ * @returns uriString that contains the formatted notebookUri
+ * If the notebookUri is untitled then we keep the query to ensure parameterized notebooks open properly
+ * otherwise we strip the query and fragment from the notebookUri for all other file schemes
+ */
+export function getNotebookUri(notebookUri: URI): string {
+	// if its untitled keep query
```

Comment:
This is clear from the code - the real thing this comment should be doing is WHY are we treating untitled special? The function comment is a little better but some actual explanation of why untitled is special would be beneficial for anyone else looking at this code.

Reply:
update the comment for better clarity